src/GafferScene/Wireframe.cpp

```diff
@@ -263,3 +263,13 @@ IECore::ConstObjectPtr Wireframe::computeProcessedObject( const ScenePath &path,
 
 	return result;
 }
+
+bool Wireframe::adjustBounds() const
+{
+	if( !Deformer::adjustBounds() )
+	{
+		return false;
+	}
+
+	return positionPlug()->getValue() != "P";
```

Comment:
width would come into account here if we decide it matters, and would probably justify the userDefault false for adjustBounds.

Reply:
This isn't the same case as the others you've noted. Here, `positionPlug()` specifies which primitive variable to use as the positions for the wireframe being generated. You can't use `width` because it's a float, and the positions need to be V3f or V2f. Note that when the position comes from "P", we're _not_ deforming, rather than the other way round.

Actually, if we want to say that "width" causes bound recompute, then we should always do a recompute for Wireframe, because it adds a "width" unconditionally, regardless of what position is in use. But as for CopyPrimitiveVariables and OSLObject, I decided that speed was more important than small adjustments for width.

Reply:
How often do caches factor width in when computing the bounds in the first place?

Reply:
Just double checked in Cortex... we don't actually implement `CurvesPrimitive::bound` so actually width doesn't affect the bound of curves...

For PointsPrimitives, we do, and it does factor into bound computations everywhere, including serialisation to SCC.